submit/templates/submit/file_process.html

```diff
@@ -38,8 +38,10 @@
       <p>What to do next:</p>
       <ul>
         <li>review the log warnings and errors</li>
-        <li>correct any missing file problems</li>
+        <li>correct any missing file problems.
+        <li>make sure links to included files match the names of the files exactly (it is case sensitive)</li>
```

Comment:
The text about "files match the names of the files exactly" is not clear. 

I suspect that user testers would not be able to explain what that sentence if we asked them about it.

I suggest detecting that a "file not found error" in the processing log matches an existing file and then showing a specific error message.

If this is a stop gap solution until we can do the work to make a error check like that, please mark this comment as resolved.

Reply:
Correct, these are quick edits we can make now as a stop-gap. They dont preclude solutions like enhancing checks and error messages that we'll continue to work on in future sprints.

Reply:
:+1: